homeassistant/components/atome/sensor.py

```diff
@@ -0,0 +1,113 @@
+"""Linky Atome."""
+import logging
+from datetime import timedelta
+from pyatome.client import AtomeClient, PyAtomeError
+import voluptuous as vol
+
+
+from homeassistant.const import CONF_USERNAME, CONF_PASSWORD, CONF_TIMEOUT, CONF_NAME
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = "atome"
+DEFAULT_UNIT = "W"
+DEFAULT_CLASS = "power"
+
+SCAN_INTERVAL = timedelta(seconds=30)
+SESSION_RENEW_INTERVAL = timedelta(minutes=55)
+DEFAULT_TIMEOUT = 10
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+        vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int,
```

Comment:
The timeout option isn't used.

Reply:
Indeed, was used before switching to a pypi package.
Fixed. Thx!